php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -1042,6 +1042,7 @@ abstract class NDB_BVL_Instrument extends NDB_Page
                  WHERE c.testID =:tid
                     AND c.pass =:cert_id
                     AND epr.centerID =:cid
+                    AND epr.active = 'Y'
```

Comment:
This is changing the query used in the instrument base class, not a query in the examiner module. This is going to break display of data for existing instruments that are using that examiner, and also make it impossible to save the inactive examiner or corrupt the data when editing old instruments that they were the examiner for.

Reply:
That said, why is this wanted in the first place? Why shouldn't the examiner module show inactive examiners (perhaps with a filter)?

Reply:
@CamilleBeau Do we have a reason for this override? I am not sure what it was in the context of LORIS Core

Reply:
Entries in this table are set to Active='N' when a site is removed for an examiner (i.e. when the box is unchecked in user accounts). There's no other way to remove an examiner from a site from the front end currently. We had this issue on CCNA because an examiner was mistakenly attributed to the wrong site, and then when it was fixed in user accounts they kept showing up in the wrong site.

Reply:
I guess it's more of a naming issue here - it's not that inactive examiners are hidden it's that the sites they are no longer active at are hidden